routes/login.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+const router = express.Router();
+const { check, validationResult } = require('express-validator');
+const bcrypt = require('bcrypt');
+
+const User = require('../models/User');
+
+const validations = [
+  check('email').trim().isEmail().normalizeEmail().withMessage('A valid email address is required'),
+  check('password').trim().escape(),
+];
+
+module.exports = () => {
+  router.get('/', (req, res) => {
+    res.render('api/user/login');
+  });
+
+  router.post('/', validations, async (req, res) => {
```

Comment:
login should be PUT or PATCH, not POST,
read about HTTP methods: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods